### PR TITLE
Plumb standalone cluster network arguments

### PR DESCRIPTION
**Pull request by stmcginnis**
This plumbs through the --pod-cidr and --service-cidr CLI arguments to
kind cluster creation. Input validation still needs to be done, but
assuming the correct CIDR format is used, clusters will be set up with
the corresponding pod and service subnets.

This also gets things in place to be able to perform other future kind
configuration customizations by populating the kind cluster struct
rather than parsing a static string.

### CODE REVIEW

**Comment by stmcginnis** [comment]
Pod CIDR setting can be verified with `kubectl get nodes -o jsonpath='{.items[*].spec.podCIDR}'` and service CIDR can be seen with `kubectl get svc -A`, and I'm sure 20 other possible ways.

File: cli/cmd/plugin/standalone-cluster/cluster/kind.go [hunk 1]
```diff
@@ -77,14 +84,50 @@ func (kcm KindClusterManager) Create(c *config.StandaloneClusterConfig) (*Kubern
...
+		kindConfig.Nodes[i].Image = c.NodeImage
+	}
+
+	// Marshal it into the raw bytes we need for creation
+	var rawConfig bytes.Buffer
+	yamlEncoder := yaml.NewEncoder(&rawConfig)
+
+	err := yamlEncoder.Encode(kindConfig)
+	if err != nil {
+		return []byte{}, fmt.Errorf("failed to generate Kind configuration. Error: %s", err.Error())
```
Comment (joshrosso) [inline cli/cmd/plugin/standalone-cluster/cluster/kind.go:113]: should we return `nil` here?

File: cli/cmd/plugin/standalone-cluster/cluster/kind.go [hunk 2]
```diff
@@ -77,14 +84,50 @@ func (kcm KindClusterManager) Create(c *config.StandaloneClusterConfig) (*Kubern
...
+	}
+	err = yamlEncoder.Close()
+	if err != nil {
+		return []byte{}, fmt.Errorf("failed to generate Kind configuration. Error: %s", err.Error())
```
Comment (joshrosso) [inline cli/cmd/plugin/standalone-cluster/cluster/kind.go:117]: should we return `nil` here?

**Review by joshrosso — CHANGES_REQUESTED** [review]
works great for me. tiny nit: Thinking we should return nil when config fails to parse. Outside of that, good to go!

**Pull request merged**